lib/growthbook/conditions.rb

```diff
@@ -23,14 +23,9 @@ def self.eval_condition(attributes, condition)
 
     # Helper function to ensure conditions only have string keys (no symbols)
     def self.parse_condition(condition)
-      case condition
-      when Array
-        return condition.map { |v| parse_condition(v) }
```

Comment:
This condition doesn't look like it can happen and tests pass when removing it, so I think it's safe to remove.